### PR TITLE
Mobile debug doc

### PR DESCRIPTION
**Pull request by yuhao900914**
# Amplitude Developer Docs PR

## Description

latest android: https://pr-834.d19s7xzcva2mw3.amplifyapp.com/data/sdks/android-kotlin/#troubleshooting-and-debugging
legacy android: https://pr-834.d19s7xzcva2mw3.amplifyapp.com/data/sdks/android/#troubleshooting-and-debugging
latest ios: https://pr-834.d19s7xzcva2mw3.amplifyapp.com/data/sdks/ios-swift/#troubleshooting-and-debugging
legacy ios: https://pr-834.d19s7xzcva2mw3.amplifyapp.com/data/sdks/ios/#troubleshooting-and-debugging

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: includes/sdk-troubleshooting-and-debugging/latest-sdk-how-to-debug.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+### How to debug
+
+Please ensure that the configuration and payload are accurate and check for any unusual messages during the debugging process. If everything appears to be right, check the value of `flushQueueSize` or `flushIntervalMillis`. Events are queued and sent in batches by default, which means they are not immediately dispatched to the server. Ensure that you have waited for the events to be sent to the server before checking for them in the charts.
+
+#### Log
+
+- Set the log level to debug to collect useful information during debuggging. [More details](./#log-level).
+- Customize `loggerProvider` class from the `LoggerProvide` and implement your own logic, like log error message in server in a a production environment. [More details](./#set-log-callback).
```
Comment (justin-fiedler) [inline includes/sdk-troubleshooting-and-debugging/latest-sdk-how-to-debug.md:8]:
```suggestion
- Customize `loggerProvider` class from the `LoggerProvider` and implement your own logic, such as logging error message in server in a production environment. [More details](./#set-log-callback).
```

File: includes/sdk-troubleshooting-and-debugging/latest-ios.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+--8<-- "includes/sdk-troubleshooting-and-debugging//latest-sdk-how-to-debug.md"
+
+#### Plugins
+You can take advantage of the Destination Plugin to print out the configuration value and event payload before sending them to the server. You can set the logLevel to debug, copy the following `ToubleShootingPlugin` into your project, add the plugin into amplitude instance.
```
Comment (justin-fiedler) [inline includes/sdk-troubleshooting-and-debugging/latest-ios.md:4]:
```suggestion
You can take advantage of a Destination Plugin to print out the configuration value and event payload before sending them to the server. You can set the logLevel to debug, copy the following `TroubleShootingPlugin` into your project, add the plugin into amplitude instance.
```

File: includes/sdk-troubleshooting-and-debugging/legacy-how-to-debug.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+### How to debug
+
+Please verify if the configuration and payload are correct and if there are any suspicious debug messages during debugging. If everything appears to be right, check the value of `eventUploadThreshold` or `eventUploadPeriodMillis`. Events are queued and sent in batches by default, which means they are not immediately dispatched to the server. Ensure that you have waited for the events to be sent to the server before checking for them in the charts.
```
Comment (justin-fiedler) [inline includes/sdk-troubleshooting-and-debugging/legacy-how-to-debug.md:3]:
```suggestion
Please verify the configuration and payload are correct and check if there are any suspicious debug messages during debugging. If everything appears to be right, check the value of `eventUploadThreshold` or `eventUploadPeriodMillis`. Events are queued and sent in batches by default, which means they are not immediately dispatched to the server. Ensure that you have waited for the events to be sent to the server before checking for them in the charts. 
```

**Review by justin-fiedler — APPROVED** [review]
Thanks @yuhao900914. See nits.

File: includes/sdk-troubleshooting-and-debugging/latest-ios.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+--8<-- "includes/sdk-troubleshooting-and-debugging//latest-sdk-how-to-debug.md"
```
Comment (Mercy811) [inline includes/sdk-troubleshooting-and-debugging/latest-ios.md:1]: extra `/` here?
```suggestion
--8<-- "includes/sdk-troubleshooting-and-debugging/latest-sdk-how-to-debug.md"
```

File: includes/sdk-troubleshooting-and-debugging/latest-android.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+--8<-- "includes/sdk-troubleshooting-and-debugging/latest-sdk-how-to-debug.md"
+
+#### Plugins
+You can take advantage of the Destination Plugin to print out the configuration value and event payload before sending them to the server. Please follow the github plugin examples to understand how it works.
```
Comment (Mercy811) [inline includes/sdk-troubleshooting-and-debugging/latest-android.md:4]: can we make this paragraph as an include file and share it with the one in latest ios?

Maybe something like
"
You can take advantage of a Destination Plugin to print out the configuration value and event payload before sending them to the server. You can set the log level to debug, copy a following Github plugin example into your project, and add the plugin into your Amplitude instance.
"

File: includes/sdk-troubleshooting-and-debugging/latest-sdk-event-callback.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+#### Event Callback
+
+The event callback will be executed after the event has been sent, for both successful and failed events. You can use this method to monitor the event status and message. You can use this way the event status and the message. Check how to set callback under **[configuration > callback](./#configuration)**.
```
Comment (Mercy811) [inline includes/sdk-troubleshooting-and-debugging/latest-sdk-event-callback.md:3]: > You can use this way the event status and the message. 
a typo or missing a word here?

Comment (yuhao900914) [reply includes/sdk-troubleshooting-and-debugging/latest-sdk-event-callback.md:3]: Duplicate with the previous sentence. Removed.

File: includes/sdk-troubleshooting-and-debugging/latest-android.md (hunk 5 above)
Comment (yuhao900914) [reply includes/sdk-troubleshooting-and-debugging/latest-android.md:4]: old ios have a different way to enable the log and there is no log level in ios...

**Pull request merged**